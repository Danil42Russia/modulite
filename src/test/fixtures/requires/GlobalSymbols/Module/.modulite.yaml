name: "@global-symbols-module"
namespace: "\\Module\\"
require:
