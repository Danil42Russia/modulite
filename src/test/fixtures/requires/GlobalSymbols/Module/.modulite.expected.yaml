name: "@global-symbols-module"
namespace: "\\Module\\"
require:
  - "\\GlobalClass"
  - "\\Common\\CommonGlobalClass"
  - "\\GlobalClass::staticMethod()"
  - "\\Common\\CommonGlobalClass::staticMethod()"
  - "\\global_function()"
  - "\\Common\\common_global_function()"
  - "\\GLOBAL_CONST"
  - "\\COMMON_GLOBAL_DEFINE"
  - "\\Common\\COMMON_GLOBAL_CONST"
  - "$GlobalVariable"
